app/src/main/java/fi/helsinki/opintoni/service/portfolio/JobSearchService.java

```diff
@@ -69,10 +69,9 @@ public JobSearchDto insert(Long portfolioId, JobSearchDto jobSearchDto) {
     }
 
     public void delete(Long portfolioId) {
-        Optional<JobSearch> jobSearchOptional = jobSearchRepository.findByPortfolioId(portfolioId);
-        if (jobSearchOptional.isPresent()) {
-            JobSearch jobSearch = jobSearchOptional.get();
+        jobSearchRepository.findByPortfolioId(portfolioId).map(jobSearch -> {
```

Comment:
turhasta `true`n palautuksesta pääsee tässä eroon niin että käyttää `.map`in sijaan `ifPresent`iä tyyliin: `jobSearchRepository.findByPortfolioId(portfolioId).ifPresent(jobSearch -> jobSearchRepository.delete(jobSearch.id))`